### DISCUSSION TITLE
編輯者的連結

### DISCUSSION THREAD

**Discussion by dannypsnl** [FizzyElt/functional-programming#58] [Ideas]
加上例如 `dannypsnl.tree`
```
\title{Lîm Tsú-thuàn}
\taxon{person}
\meta{external}{https://dannypsnl.me}
\meta{institution}{second state}
\meta{position}{Compiler Developer}
```
這樣 `\author{dannypsnl}` 就會參照這張卡片了

**Comment by FizzyElt** [comment]
可以單獨分資料夾出來存嗎，還是只能直接丟進 trees 資料夾理

**Comment by dannypsnl** [comment]
在 `trees` 底下的任何 subdirectory 都可以，另外一種方法就是下 `forester build` 指令的時候多給其他目錄，e.g. `forester build ... trees/ people/`

兩種應該都可以

**Comment by FizzyElt** [comment]
目前應該是 trees 裡面開 subdirectory 存放，未來可能會把不同主題用 subdirectory 分類

**Reply by dannypsnl** [reply]
ok
